extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/PausedMethodTest.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.quartz.test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class PausedMethodTest {
+
+    @Inject
+    Scheduler quartzScheduler;
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(Jobs.class));
+
+    private static final String IDENTITY = "myScheduled";
+
+    @Test
+    public void testSchedulerPauseMethod() throws InterruptedException {
+        quartzScheduler.pause(IDENTITY);
+        assertFalse(PausedMethodTest.Jobs.LATCH.await(3, TimeUnit.SECONDS));
```

Comment:
This assert expects that `Jobs.countDownSecond()` is invoked at most once before the test method is executed and although it should happen in 99% cases (at least locally) it's not deterministic. I wonder if we could improve the test somehow...

Reply:
Maybe it helps when I disable scheduler by default and start it with the test? Not sure if that makes it more deterministic.

Reply:
> Maybe it helps when I disable scheduler by default and start it with the test? Not sure if that makes it more deterministic.

Ok that's not working, I cannot enable the scheduler programmatic

Reply:
@adrianfiedler  Looks like this test is failing in CI. See https://github.com/quarkusio/quarkus/pull/16981#issuecomment-832547361. Would you have some time to have a look?

Reply:
I can take a look right now.

Reply:
https://github.com/quarkusio/quarkus/pull/17003 should fix the problem.